kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/clients/KcatClient.java

```diff
@@ -83,8 +84,8 @@ public void produceMessages(String topicName, String bootstrap, String message,
     @Override
     public List<ConsumerRecord> consumeMessages(String topicName, String bootstrap, int numOfMessages, Duration timeout) {
         LOGGER.atInfo().log("Consuming messages using kcat");
-        String name = Constants.KAFKA_CONSUMER_CLIENT_LABEL + "-kcat";
-        List<String> args = List.of("-b", bootstrap, "-K ,", "-t", topicName, "-C", "-c", String.valueOf(numOfMessages), "-e", "-J");
+        String name = Constants.KAFKA_CONSUMER_CLIENT_LABEL + "-kcat-" + MobyNamesGenerator.getRandomName().replace("_", "-");
+        List<String> args = List.of("-b", bootstrap, "-K ,", "-t", topicName, "-C", "-o", "-" + numOfMessages, "-e", "-J");
```

Comment:
This is another way to get the latest N messages that works better when we are in the case we are reading from the same topic with a different consumer and we want to avoid reading again the same message.

In kcat we don't have the way to commit the messages or use a consumer group...

Reply:
Ok, that makes sense can you pop a comment next to the command noting that (odds are I'll forget it otherwise)